examples/plot_vector_testing.py

```diff
@@ -0,0 +1,205 @@
+#
+# .. note::
+#   Each geometry type must be tested seperately in VectorTester. So, if your
+#   plot has multiple geometry types, such as lines, polygons, and points,
+#   make sure to check each geometry type separately.
+
+###############################################################################
+#
+# .. note::
+#   Most tests are created as assert statements. Thus, if a test fails,
+#   matplotcheck will return an error. If the test passes, no message is
+#   returned.
+
+################################################################################
+# Test Point Attribute Values and Geometry xy Locations
+# -----------------------------------------------------
+# You can check that both the position of the points on the plot and the
+# associated point attribute values are
+# accurate using assert_points(), assert_points_grouped_by_type() and
+# assert_collection_sorted_by_markersize().
+#
+# To check the geometry locations, you can call assert_points() and prove the
+# expected points data, which in this case is the point_gdf object.
+#
+# If the plot uses point markers that are sized by attribute value, you can
+# check that the size of each marker correctly relates to an attribute value by
+# providing the geometry, point_gdf here, and the attribute the size is based
+# off of, which is the 'size' column in this case.
+
+
+
+# Check point geometry location (x, y location)
+vector_test.assert_points(point_gdf)
+
+# Check points are grouped plotted by type
+vector_test.assert_points_grouped_by_type(point_gdf, "size")
+
+# Check points size is relative to a numeric attribute value
+vector_test.assert_collection_sorted_by_markersize(point_gdf, "size")
+
+################################################################################
+# Test Line Attribute Values and Coordinate Information (x, y)
+# ------------------------------------------------------------
+# You can also test the position and plot values of line
+# geometries.
+
+# Check line geometry
+vector_test.assert_lines(multi_line_gdf)
+
+# Check lines plotted by type
+vector_test.assert_lines_grouped_by_type(multi_line_gdf, "attr")
+
+################################################################################
+# Testing Polygon Geometry
+# ------------------------
+# Currently, MatPlotCheck is unable to check that polygons were plotted by type.
+# Eventually this will be supported. For the time being though, you can still
+# check that polygons are plotted correctly!
+
+# Check Polygons
+vector_test.assert_polygons(polygon_gdf)
+
+################################################################################
+# Access the Axes object in a Jupyter Notebook
+# --------------------------------------------
+# MatPlotCheck can be used to help grade Jupyter Notebooks as well. The main
+# difference is in how you would store the Axes from the plot you are grading.
+# Below is an example of how you could store the Axes of a plot you are hoping
```

Comment:
"Below is an example of how you could access the axes of a plot you want to test in a Jupyter Notebook."